src/verify.rs

```diff
@@ -1,26 +1,9 @@
 use bit_set::Bitset;
-use diag::Diagnostic;
-use nameck::Atom;
-use nameck::Nameset;
-use parser;
-use parser::Comparer;
-use parser::copy_token;
-use parser::NO_STATEMENT;
-use parser::Segment;
-use parser::SegmentId;
-use parser::SegmentOrder;
-use parser::SegmentRef;
-use parser::StatementAddress;
-use parser::StatementRef;
-use parser::StatementType;
-use parser::TokenPtr;
-use scopeck::ExprFragment;
-use scopeck::Frame;
-use scopeck::Hyp;
-use scopeck::ScopeReader;
-use scopeck::ScopeResult;
-use scopeck::ScopeUsage;
-use scopeck::VerifyExpr;
+use diag::{Diagnostic, Result};
```

Comment:
I use single-name imports exclusively because they can be sorted with editor commands.  If and when rustfmt starts sorting imports by default, I'll switch to group imports.